collections/misc/collbackup.php

```diff
@@ -1,25 +1,29 @@
 <?php
 include_once('../../config/symbini.php');
-include_once($SERVER_ROOT.'/content/lang/collections/misc/collbackup.'.$LANG_TAG.'.php');
-header("Content-Type: text/html; charset=".$CHARSET);
+if($LANG_TAG != 'en' && file_exists($SERVER_ROOT . '/content/lang/collections/misc/collbackup.' . $LANG_TAG . '.php')){
+	include_once($SERVER_ROOT . '/content/lang/collections/misc/collbackup.' . $LANG_TAG . '.php');
+}
+else include_once($SERVER_ROOT . '/content/lang/collections/misc/collbackup.en.php');
+header('Content-Type: text/html; charset=' . $CHARSET);
 
-$collid = array_key_exists("collid",$_REQUEST)?$_REQUEST["collid"]:0;
-$action = array_key_exists("formsubmit",$_REQUEST)?$_REQUEST["formsubmit"]:'';
-$cSet = array_key_exists("cset",$_REQUEST)?$_REQUEST["cset"]:'';
+$collid = isset($_REQUEST['collid']) ? filter_var($_REQUEST['collid'], FILTER_SANITIZE_NUMBER_INT) : 0;
+$action = isset($_REQUEST['formsubmit']) ? $_REQUEST['formsubmit'] : '';
+$cSet = isset($_REQUEST['cset']) ? $_REQUEST['cset'] : '';
```

Comment:
Same here. If you agree with the above, implement `htmlspecialchars($foo, HTML_SPECIAL_CHARS_FLAGS)`.